snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/validation/SnomedQueryValidationRuleEvaluator.java

```diff
@@ -63,17 +67,24 @@
 	private static final TypeReference<SnomedComponentValidationQuery<?, PageableCollectionResource<SnomedComponent>, SnomedComponent>> TYPE_REF = new TypeReference<SnomedComponentValidationQuery<?, PageableCollectionResource<SnomedComponent>, SnomedComponent>>() {};
 
 	@Override
-	public List<ComponentIdentifier> eval(BranchContext context, ValidationRule rule) throws Exception {
+	public List<IssueDetail> eval(BranchContext context, ValidationRule rule) throws Exception {
 		checkArgument(type().equals(rule.getType()), "'%s' is not recognizable by this evaluator (accepts: %s)", rule, type());
 		return context.service(ObjectMapper.class)
 				.<SnomedComponentValidationQuery<?, PageableCollectionResource<SnomedComponent>, SnomedComponent>>readValue(rule.getImplementation(), TYPE_REF)
 				.prepareSearch()
 				.all() // always return all hits
-				.setFields(SnomedComponentDocument.Fields.ID)
+				.setFields(SnomedComponentDocument.Fields.ID, SnomedComponentDocument.Fields.MODULE_ID, SnomedComponentDocument.Fields.ACTIVE)
 				.build()
 				.execute(context)
 				.stream()
-				.map(SnomedComponent::getComponentIdentifier)
+				.map(component -> {
+					final ComponentIdentifier affectedComponent = component.getComponentIdentifier();
+					final String moduleId = component.getModuleId();
+					final Map<String, Object> details = newHashMap();
+					details.put(SnomedRf2Headers.FIELD_MODULE_ID, moduleId);
+					details.put(SnomedRf2Headers.FIELD_ACTIVE, Boolean.valueOf(component.isActive()));
```

Comment:
`component.isActive()` already returns a wrapper object, no need for `Boolean.valueOf()` call.